kong/kong.lua

```diff
@@ -144,9 +144,11 @@ function Kong.init()
 end
 
 function Kong.init_worker()
+  prepare_module_variables()
```

Comment:
You're right that the upvalues follow into `init_worker`. This was left over from an experiment to generate a new set of the values in `init_worker`, which reveals the [`API disabled in the context of init_worker_by_lua*`](https://github.com/Mashape/kong/issues/966) errors.

Removing this now, avoiding any functional change with this PR.